cmd/gokr-packer/packer.go

```diff
@@ -6,18 +6,19 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/gokrazy/internal/httpclient"
+	"github.com/gokrazy/internal/updater"
```

Comment:
nit: for consistency, can you please move all github imports to a separate import group (separated by a blank line)?